ionics_fits/models/sinusoid.py

```diff
@@ -268,6 +268,8 @@ def estimate_parameters(self, x: TX, y: TY):
             setattr(sine.parameters["omega"], attr_name, attr_value_pr)
 
         for attr_name in attrs:
+            if attr_name in ["lower_bound", "upper_bound"]:
+                continue
```

Comment:
I'm inclined to say we should have two `attr` lists, so something like:

```python
phi_attrs = ("fixed_to", "user_estimate")
attrs = (*phi_attrs, "lower_bound", "upper_bound")
```

Reply:
Implemented in https://github.com/OxIonics/ionics_fits/pull/176/commits/3167fa28c6f46ffb2b4e9df9f4dfc1360d292688